src/app/shared/footer/footer.scss

```diff
@@ -1,6 +1,50 @@
 .docs-footer {
   margin-top: 40px;
   padding: 15px;
-  text-align: center;
   font-size: 12px;
 }
+
+.footer-list {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  padding: 8px;
+}
+
+.docs-angular-logo {
+  height: 50px;
+}
+
+.footer-logo {
+  width: 8.33%;
+}
+
+.footer-cc {
+  display: flex;
+  flex: 1;
+  justify-content: flex-end;
+}
+
+footer ul {
```

Comment:
Can these styles use the css classes instead? There's already a class on the `ul`